src/repl/scala/tools/nsc/interpreter/MemberHandlers.scala

```diff
@@ -136,20 +136,28 @@ trait MemberHandlers {
           if (mods.isLazy) codegenln(false, "<lazy>")
           else any2stringOf(path, maxStringElements)
 
-        val nameString = string2code(prettyName) + (if (showObjIds) s"""" + f"@$${System.identityHashCode($path)}%8x" + """" else "")
+        val varOrValOrLzy =
+          if (mods.isMutable) "var"
+          else if (mods.isLazy) "lazy val"
+          else "val"
+
+        val nameString = {
+          string2code(prettyName) + (
+            if (showObjIds) s"""" + f"@$${System.identityHashCode($path)}%8x" + """"
```

Comment:
does `symbol.id` serve a similar purpose?

Reply:
This one I just reformatted to fit on one line, but this is outputting code to get the hashcode of the actual runtime object I think

Reply:
Yeah. I'm just wondering if it'd be better to use the existing `id` field (tending to be smaller)